src/components/Footer.js

```diff
@@ -7,10 +7,12 @@ import Image from 'react-bootstrap/Image';
 class Footer extends Component{
 	render(){
 		return (
-			<Row>
-				 <Col xs={12} sm={12} md={12} lg={12} xl={12}>
-					<div id="footer">
-						<div id="footer_social_links_container">
+				<footer id="footer">
+					<Row>
+						<Col xs={12} sm={12} md={6} className="text-center text-md-start order-last order-md-first mt-5px" id="copyright_links">
+							© 2021 xSurge. All rights reserved.
```

Comment:
So it centers on mobile first. But on medium screens, it's left aligned (text-md-start). We flip the the order of elements between text and social media icons, but flip the order the other way around on mobile. That's order-last vs order-md-first